.github/workflows/ci.yml

```diff
@@ -16,5 +16,5 @@ jobs:
       - uses: actions/setup-go@v3
       - name: Run Unit Tests
         run: |
-          export SSH_INTEGRATION_SEMANTIC="${{secrets.SSH_INTEGRATION_SEMANTIC}}"
+          export SSH_INTEGRATION_SEMANTIC="${{ secrets.SSH_INTEGRATION_SEMANTIC }}"
```

Comment:
I think we can remove this line, since we are using user/pass, right? @esequielvirtuoso

Reply:
Done!